packages/core/src/index.ts

```diff
@@ -1,21 +1,11 @@
-import {
-  after,
-  done,
-  doneInvoke,
-  escalate,
-  forwardTo,
-  respond,
-  sendParent,
-  sendTo
-} from './actions.ts';
-import { assign } from './actions/assign.ts';
-import { cancel } from './actions/cancel.ts';
-import { choose } from './actions/choose.ts';
-import { log } from './actions/log.ts';
-import { pure } from './actions/pure.ts';
-import { raise } from './actions/raise.ts';
-import { send } from './actions/send.ts';
-import { stop } from './actions/stop.ts';
+import { doneInvoke, forwardTo, sendParent, sendTo } from './actions.ts';
+export { assign } from './actions/assign.ts';
+export { cancel } from './actions/cancel.ts';
+export { choose } from './actions/choose.ts';
+export { log } from './actions/log.ts';
+export { pure } from './actions/pure.ts';
+export { raise } from './actions/raise.ts';
+export { stop } from './actions/stop.ts';
```

Comment:
With those root exports here we kinda don't need the separate entry point:
https://github.com/statelyai/xstate/blob/d46e0399cecc7c4a4973686a4ef1d74530fe0e1d/packages/core/package.json#L17-L20

I think that there is no value in keeping those exports available in both places, we should just stick to one of them. Which one? I don't have strong opinion about this really, a separate entrypoint might make it a little bit easier to treeshake at times - but I know how to do just anything treehskaeable so it's not a big issue anyway. Let's just think about what is better for the consumer.

Reply:
Pros for index:

- Easier migration (devs don't need to do anything from v4 -> v5)

Pros for /actions:

- ~~More tree-shakeable? (maybe not)~~

Reply:
I removed this; can you check to see if I've done it correctly?